hack/generate-ta-tasks.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o posix
```

Comment:
Just out of curiosity, why do you enable the posix option?

Reply:
I think it was because of the exit code from subshells (or something akin to that), might be required in the previous version when this was included in `hack/build-and-push.sh`.

There is a list of changes in behavior when in the posix mode here: https://www.gnu.org/software/bash/manual/html_node/Bash-POSIX-Mode.html